### ISSUE TITLE
taint: remove false-positive taint source from Request::integer() and Request::float()

### ISSUE THREAD

**Issue by alies-dev** [psalm/psalm-plugin-laravel#574]
## Problem

`Request::integer()` and `Request::float()` in `InteractsWithInput` are currently marked as `@psalm-taint-source input`, but their implementations use `(int)` and `(float)` casts respectively:

```php
// Laravel source
public function integer($key, $default = 0): int {
    return (int) $this->input($key, $default);
}

public function float($key, $default = 0.0): float {
    return (float) $this->input($key, $default);
}
```

A numeric cast **cannot** produce an injection payload:
- `(int) "1; DROP TABLE users"` → `1`
- `(float) "<script>alert(1)</script>"` → `0.0`

This is the same reasoning that already excludes `boolean()`, `date()`, and `enum()` from being taint sources in the same stub file.

## Fix

In `stubs/common/Http/Concerns/InteractsWithInput.stubphp`, remove `@psalm-taint-source input` from `integer()` and `float()`.

## Stub file

`stubs/common/Http/Concerns/InteractsWithInput.stubphp`
